app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -122,55 +208,81 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        :association => :storages,
+        # TODO: change :manager_ref => [:location],
+        :association => :storages
```

Comment:
@pkliczewski yes the ovirt parser sets the location for the storage here:
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/refresh/parse/parser.rb#L40-L44
If that NFS store was attached to hosts in different ext_management_systems it would get the same location and be linked to the first